modules/__tests__/enhancer-test.js

```diff
@@ -3,7 +3,8 @@ var Enhancer = require('inject!enhancer.js')({
   './resolve-styles.js': resolveStyles
 });
 
-var {Component} = require('react');
+var React = require('react/addons');
```

Comment:
we should probably be consistent and always require react/addons in tests, might help a bit with speed (just conjecture).